packages/atomic/src/components.d.ts

```diff
@@ -611,6 +621,16 @@ export namespace Components {
     }
     interface AtomicResultFieldsList {
     }
+    interface AtomicResultHtml {
+        /**
+          * The result field which the component should use. This will look in the Result object first, and then in the Result.raw object for the fields. It is important to include the necessary field in the ResultList component.
```

Comment:
```suggestion
          * The result field which the component should use.
          * If set, Atomic searches for the specified field in the `Result` object first.
          * If there's no such a field, Atomic searches throught the `Result.raw` object.
          * It's important to include the necessary field in the `ResultList` component.
```

For this and other occurrences of this comment🙏